Makefile

```diff
@@ -49,6 +49,8 @@ ci: batch
 # Run Ansible from the virtualenv.
 ansible-playbook-run-%: ve
 	OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES \
+	JUNIT_TASK_CLASS=yes \
+	JUNIT_OUTPUT_DIR=logs \
```

Comment:
Treat the ansible tasks as JUnit test results to honor the JUnit reporting in the CI, then it will be easy to visualise errors related to the ansible execution as test failures.

The GitHub PR comment will also honor those test results in the comment:
- https://github.com/elastic/beats-tester/pull/157#issuecomment-651570307